Sources/Brave/WebFilters/AdBlockEngineManager.swift

```diff
@@ -184,41 +184,39 @@ public actor AdBlockEngineManager: Sendable {
 }
 
 #if DEBUG
-extension AdBlockEngineManager {
+private extension AdBlockEngineManager {
   /// A method that logs info on the given resources
-  fileprivate func debug(compiledResults: [ResourceWithVersion: Result<Void, Error>]) {
-    let resourcesString = compiledResults.sorted(by: { $0.key.order < $1.key.order })
-      .map { (resourceWithVersion, compileResult) -> String in
+  func debug(compiledResults: [ResourceWithVersion: Result<Void, Error>]) {
+    log.debug("Loaded \(compiledResults.count, privacy: .private) (total) engine resources:")
```

Comment:
As mentioned `privacy: .private` is redundant

Reply:
Fixed